conda_forge_tick/update_upstream_versions.py

```diff
@@ -20,18 +20,18 @@
 from typing import Any, Iterable
 
 # conda_forge_tick :: cft
-logger = logging.getLogger("conda-forge-tick._update_versions")
+logger = logging.getLogger("conda-forge-tick.update_upstream_versions")
 
 
 def get_latest_version(
     name: str,
     meta_yaml: Any,
     sources: Iterable[AbstractSource],
 ) -> dict:
-    version_data = {}
-    # avoid
+    version_data = {"new_version": False}
```

Comment:
@viniciusdc I set things to False at the top

Reply:
Oh, Great!

Reply:
isn't it better to set `version_data = {}` from the beginning, then modify the else to elif and then set an else for `False` condition?... (this way we don't have the problem CJ pointed)

Reply:
It doesn't matter. As long as we only override the version in the graph nodes if it is newer or the graph nodes don't have a valid version, then it will work.